frontend/src/component/onboarding/ConnectionInformation.tsx

```diff
@@ -0,0 +1,109 @@
+import { styled, Typography, useTheme } from '@mui/material';
+import { ConditionallyRender } from '../common/ConditionallyRender/ConditionallyRender';
+import { WhitePulsingAvatar } from '../project/Project/Import/PulsingAvatar';
+import Pending from '@mui/icons-material/Pending';
+import useProjectOverview from 'hooks/api/getters/useProjectOverview/useProjectOverview';
+import { useEffect } from 'react';
+
+interface IConnectionInformationProps {
+    onConnection: () => void;
+    projectId: string;
+}
+export const Container = styled('div')(({ theme }) => ({
+    backgroundColor: theme.palette.background.sidebar,
+    padding: theme.spacing(6, 9, 6, 9),
+    minWidth: '400px',
+    display: 'flex',
+    flexDirection: 'column',
+    color: theme.palette.primary.contrastText,
+}));
+
+export const Title = styled(Typography)(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    fontWeight: theme.typography.fontWeightBold,
+}));
+
+export const SdkInfo = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'row',
+    padding: theme.spacing(4, 0, 12, 0),
+    justifyContent: 'space-between',
+    fontSize: theme.spacing(1),
+}));
+
+export const Info = styled('div')(({ theme }) => ({
+    display: 'flex',
+    gap: theme.spacing(1),
+    flexDirection: 'column',
+}));
+
+export const ConnectionStatus = styled('div')(({ theme }) => ({
+    alignItems: 'center',
+    justifyContent: 'center',
+    display: 'flex',
+    gap: theme.spacing(2),
+    flexDirection: 'column',
+    fontSize: theme.fontSizes.smallBody,
+}));
+
+export const ConnectionInformation = ({
+    onConnection,
+    projectId,
+}: IConnectionInformationProps) => {
+    const theme = useTheme();
+    const { project } = useProjectOverview(projectId, {
+        refreshInterval: 1000,
+    });
+
+    const onboarded = project.onboardingStatus.status === 'onboarded';
+
+    useEffect(() => {
+        if (onboarded) {
+            onConnection();
```

Comment:
did you think about the scenario where people come back to the 3rd stage again? E.g. after closing the dialog and opening it again? Or when going back and forth between the stages?